src/main/features/response/views/confirmation.njk

```diff
@@ -18,10 +18,17 @@
         <p>
           <span class="bold">{{ claim.respondedAt | date }}</span>
         </p>
-        <p class="receipt-download-container">
-          {{ internalLink(t('Download your response'), ResponsePaths.receiptReceiver.evaluateUri({ externalId: claim.externalId })) }}
-          {{ t('(PDF)') }}
-        </p>
+        {% if claim.response.directionsQuestionnaire.selfWitness %}
+          <p class="receipt-download-container">
+            {{ internalLink(t('Download your response and hearing requirements'), ResponsePaths.receiptReceiver.evaluateUri({ externalId: claim.externalId })) }}
+            {{ t('(PDF)') }}
+          </p>
+        {% else %}
+          <p class="receipt-download-container">
+            {{ internalLink(t('Download your response'), ResponsePaths.receiptReceiver.evaluateUri({ externalId: claim.externalId })) }}
+            {{ t('(PDF)') }}
```

Comment:
Looks like both are same hyperlink, not sure if above `if` condition is necessary as pdf is generated based on whatever we have saved in backend.

Reply:
as agreed ... no code change